terms/syntheticfeatures_fcm.bib

```diff
@@ -0,0 +1,57 @@
+@inproceedings{pedreschi2008,
+  author = {Pedreschi, Dino and Ruggieri, Salvatore and Turini, Franco and Pedreshi, Dino and Ruggieri, Salvatore and Turini, Franco},
+  booktitle = {Proceedings of the ACM SIGKDD International Conference on Knowledge Discovery and Data Mining},
+  doi = {10.1145/1401890.1401959},
+  isbn = {9781605581934},
+  issn = {0309-0167 (Print)},
+  keywords = {,classification rules,discrimination},
+  number = {May 2014},
+  pages = {560--568},
+  pmid = {7698734},
+  title = {{Discrimination-aware data mining}},
+  url = {http://dl.acm.org/citation.cfm?doid=1401890.1401959},
```

Comment:
Did you read the instructions in the README file ???

> .bib references do not need DOI, keywords or abstract entries (other attributes to delete are Remove Date-Added, Date-Modified, Bdsk-File-1)

Also remove url.  Not needed.